src/command/manifest/download.ts

```diff
@@ -13,6 +13,8 @@ export class Download extends ManifestCommand implements LeafCommand {
   public readonly name = 'download'
   public readonly description = 'Download manifest content to a folder'
 
+  public address!: BzzAddress
```

Comment:
well, it seems a bit interesting how you juggle with the `address` field between commands, but I suggest that you should define it in the `run` method as an optional parameter.